lib/core/src/lib/ui-header/ui-header.component.html

```diff
@@ -0,0 +1,41 @@
+<ng-container [ngSwitch]="variant">
+    <ng-container *ngSwitchCase="'minimal'">
+        <adf-toolbar [color]="color">
+            <adf-toolbar-title>
+                <ng-container *ngTemplateOutlet="toolbarTitleContent"></ng-container>
+            </adf-toolbar-title>
+            <ng-container *ngTemplateOutlet="navbarTemplate"></ng-container>
+            <div class="adf-toolbar--spacer"></div>
+            <ng-container *ngTemplateOutlet="toolbarActions"></ng-container>
+        </adf-toolbar>
+    </ng-container>
+
+    <ng-container *ngSwitchCase="'extended'">
+        <adf-toolbar [color]="color">
+            <adf-toolbar-title>
+                <ng-container *ngTemplateOutlet="toolbarTitleContent"></ng-container>
+            </adf-toolbar-title>
+            <ng-container *ngTemplateOutlet="toolbarActions"></ng-container>
+        </adf-toolbar>
+        <adf-toolbar [color]="color">
+            <ng-container *ngTemplateOutlet="navbarTemplate"></ng-container>
+        </adf-toolbar>
+    </ng-container>
+</ng-container>
+
+<ng-template #toolbarTitleContent>
+    <img *ngIf="logoSrc" [src]="logoSrc" [alt]="logoAlt" class="adf-toolbar-logo"/>
+    <span *ngIf="variant==='extended'">{{ title | translate }}</span>
+</ng-template>
+
+<ng-template #toolbarActions>
+    <ng-content select=".adf-toolbar-actions"></ng-content>
```

Comment:
Is it necessary to use a class here? Can we use something else like dummy attributes? Something like 
```
<ng-content select="[adf-toolbar-actions]"></ng-content>
```
and then
```
<div adf-toolbar-actions ...></div>
```
Mandating a class seems kind of strange. This change would however impact the customisability (theming) of this component

Reply:
Thanks, looks much better now.